docs/Installation/Installation.md

```diff
@@ -8,6 +8,11 @@ This page details the installation procedure for SpECTRE on personal computers.
 For instructions on installing SpECTRE on clusters please refer to the \ref
 installation_on_clusters "Installation on clusters" page.
 
+### Quick-start guide for code development
```

Comment:
I don't think this should be here. VSCode is not an editor we are endorsing or anything like that. I'm fine with a `Dev Guide` entry on "Building SpECTRE with VSCode and Docker", but it should **not** make it sound like VSCode or that work flow is at all the recommended workflow.

Reply:
I agree in general...no one is telling you or others to switch! But it is an especially easy way to get started for new developers, so I think it should be ok to say it's something new devs might want to consider, especially if they aren't experienced setting up their own dev environment, along with language that they can use tools of their choice if they wish and that we don't endorse or preferentially support any particular choice